org.gnome.Epiphany.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id" : "org.gnome.Epiphany",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "epiphany",
+    "finish-args" : [
+        "--device=dri",
+        "--filesystem=/run/.heim_org.h5l.kcm-socket",
```

Comment:
This is new whats that about? Also you surely need `:create` so it doesn't fail when it doesn't exist?

Reply:
> This is new whats that about? Also you surely need `:create` so it doesn't fail when it doesn't exist?
This comes from upstream's manifest https://gitlab.gnome.org/GNOME/epiphany/-/blob/master/org.gnome.Epiphany.json#L13

Reply:
It's Kerberos, let's you sign into Fedora and RH infrastructure without typing your password.

Reply:
I don't think :create would work btw, it's a socket, needs to be connected to something on the host ;)